packages/filesystem/src/node/encoding-util.ts

```diff
@@ -0,0 +1,54 @@
+/********************************************************************************
+ * Copyright (C) 2019 Xuye Cai and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+// copied from vscode: https://github.com/Microsoft/vscode/blob/master/src/vs/base/node/encoding.ts
```

Comment:
@vince-fugnitto I received an email notification about probably needing a CQ for this, but I can't find the comment here. What was the conclusion?

Reply:
I believe the initial CQ for the PR did not cover these files (and potentially others in the PR).
I guess we should change our ways of working a little to only register CQs when a PR is completed and ready to be merged as not to create intermediate CQs.

Reply:
@vince-fugnitto since @marcdumais-work is away could you take care of ip issues for this PR?

Also @vince-fugnitto @tolusha would it be alright if i add you to ip-team as well? I think you are familiar enough with the process.

Reply:
@akosyakov I don't mind.

Reply:
@akosyakov I'm fine as well being part of the ip-team.